zingolib/src/lightclient/send.rs

```diff
@@ -133,16 +133,18 @@ impl LightClient {
 #[cfg(feature = "zip317")]
 /// patterns for newfangled propose flow
 pub mod send_with_proposal {
-    use std::convert::Infallible;
+    use std::{convert::Infallible, ops::DerefMut as _};
```

Comment:
can we just use DerufMut without the _?

Reply:
We can, I like the convention of using `_` for traits that aren't named in the file, to indicate that they are imported for side effects (i.e. getting their trait methods into scope).